Client/src/Components/DataCell/DataCell.js

```diff
@@ -87,8 +94,24 @@ export default function DataCell() {
                             <input type="file" id="jsonFile" accept="application/json" />
                             <Button type="submit" variant="contained"> Upload this file! </Button>
                         </div>
+
                     </form>
                 </Box>
+
+                <Box  p={4}>
+                    <div>
+                        {/**for downloading sample empty json file*/}
+                         <Download file= "emptyJason.json" content= {JSON.stringify(emptyJSFile,null,2)}>
+                            <Button type="submit" variant="contained" onClick={console.log('success fully json downloaded')}> Download Sample JSON file </Button>
+                         </Download>                                                               
+                    </div>
+                    {/**for downnloading instructions readMe for users */}
+                    <div>
+                        <Download file="readMe.txt" content = {readme + fields + info}>

```

Comment:
Maybe changing the ReadMe button to read "Download json file instructions" would be easier for the user experience, I didn't know what the link was until I read the file downloaded

Reply:
changed it to download json file submission instructions. thanks for the feedback